### PR TITLE
added alt text to the images

### PR DESCRIPTION
**Pull request by Lhorla001**
#### Summary

I added `alt` attributes to 5 images in the **what is javascript** content
#### Motivation

`alt` attributes improve accessibility
#### Supporting details

N/A
#### Related issues

N/A
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/javascript/first_steps/what_is_javascript/index.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ The three layers build on top of one another nicely. Let's take a simple text la
 <p>Player 1: Chris</p>
 ```
 
-![](just-html.png)
+![Paragraph of Player 1: Chris](just-html.png)
```
Comment (estelle) [inline files/en-us/learn/javascript/first_steps/what_is_javascript/index.md:59]:
```suggestion
![Paragraph of Player 1: Chris as plain text](just-html.png)
```

File: files/en-us/learn/javascript/first_steps/what_is_javascript/index.md [hunk 2]
```diff
@@ -114,7 +114,7 @@ They do the same thing for programming that ready-made furniture kits do for hom
 
 They generally fall into two categories.
 
-![](browser.png)
+![A screenshot showing the two categories of API; 3rd party APIs and browser APIs](browser.png)
```
Comment (estelle) [inline files/en-us/learn/javascript/first_steps/what_is_javascript/index.md:117]:
```suggestion
![Two categories of API; 3rd party APIs are shown to the side of the browser and browser APIs are in the browser](browser.png)
```

File: files/en-us/learn/javascript/first_steps/what_is_javascript/index.md [hunk 3]
```diff
@@ -144,7 +144,7 @@ Here we'll actually start looking at some code, and while doing so, explore what
 
 Let's briefly recap the story of what happens when you load a web page in a browser (first talked about in our [How CSS works](/en-US/docs/Learn/CSS/First_steps/How_CSS_works#how_does_css_actually_work) article). When you load a web page in your browser, you are running your code (the HTML, CSS, and JavaScript) inside an execution environment (the browser tab). This is like a factory that takes in raw materials (the code) and outputs a product (the web page).
 
-![](execution.png)
+![An image showing how the code comes togther and shows on the browser tab when the page is loaded](execution.png)
```
Comment (estelle) [inline files/en-us/learn/javascript/first_steps/what_is_javascript/index.md:147]:
```suggestion
![HTML, CSS and JavaScript code come together to create the content in the browser tab when the page is loaded](execution.png)
```

File: files/en-us/learn/javascript/first_steps/what_is_javascript/index.md [hunk 4]
```diff
@@ -395,7 +395,7 @@ They won't run until the page content has all loaded, which is useful if your sc
 
 Here is a visual representation of the different script loading methods and what that means for your page:
 
-![](async-defer.jpg)
+![An image showing how the three script loading method works](async-defer.jpg)
```
Comment (estelle) [inline files/en-us/learn/javascript/first_steps/what_is_javascript/index.md:398]:
```suggestion
![How the three script loading method work: default has parsing blocked while JavaScript is fetched and executed. With async, the parsing pauses for execution only. With defer, parsing isn't paused, but execution on happens after everything is else is parsed.](async-defer.jpg)
```

**Review by estelle — APPROVED** [review]
Congratulations on your first PR!

**Comment by Lhorla** [comment]
Thank you, Estelle.

**Pull request merged**